src/index.html

```diff
@@ -2,15 +2,52 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://127.0.0.1:5500/src"
+      class="search big"
+      method="post"
+      data-qa="big"
+    >
+      <button
+        type="submit"
+        class="button button-big"
+        data-qa="keypress"
+      >
+      </button>
+
+      <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        name="search"
+        class="input input-big"
+      >
+    </form>
+
+    <form
+      action="#"
+      class="search small"
+      method="post"
+      data-qa="small"
+    >
+      <button
+        type="submit"
+        class="button button-small"
+        data-qa="keypress"
+      >
+      </button>
+      
+      <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        name="search"
+        class="input input-small"
+      >
+    </form>
```

Comment:
```suggestion
      class="search"
      method="post"
      data-qa="big"
    >
      <button
        type="submit"
        class="button"
        data-qa="keypress"
      >
      </button>

      <input
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        name="search"
        class="input"
      >
    </form>

    <form
      action="#"
      class="search small"
      method="post"
      data-qa="small"
    >
      <button
        type="submit"
        class="button button-small"
        data-qa="keypress"
      >
      </button>
      
      <input
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        name="search"
        class="input input-small"
      >
    </form>
```

let make it work like this